configure.ac

```diff
@@ -101,8 +106,8 @@ dnl Check for gapi
 AC_PATH_PROG(GAPI_PARSER, gapi2-parser, no)
 if test "x$GAPI_PARSER" = "xno"; then
 	AC_MSG_CHECKING(for gapi2-parser.exe)
-	GAPI_PARSER=`which gapi2-parser.exe 2> /dev/null`
```

Comment:
If you want, I can replace the `which` calls with `find` calls which look for the .exe file inside $prefix. (But that would go in a different PR.)